src/Sidekick.Business/Parsers/Models/Rarity.cs

```diff
@@ -2,14 +2,14 @@ namespace Sidekick.Business.Parsers.Models
 {
     public enum Rarity
     {
-        Unknown, // Gem, Currency, Divination Card, Quest Item, Prophecy, Relic, etc.
-        Currency,
-        Gem,
-        DivinationCard,
+        Normal = 0,
+        Magic = 1,
+        Rare = 2,
+        Unique = 3,
+        Gem = 4,
+        Currency = 5,
+        DivinationCard = 6,
 
-        Normal,
-        Magic,
-        Rare,
-        Unique,
+        Unknown = 99,
```

Comment:
For enums it is usually a good practice to have the default, unknown case be 0 rather than another value. It makes more sense that `default(Rarity)` would return `Unknown` rather than `Normal`.

Reply:
I agree with you, but I am limited in my options as I need Normal to be 0, because of the official API.

The frameType property of the item results uses these values:
 Possible values with examples:
              * `0` - normal items
              * `1` - magic items
              * `2` - rare items
              * `3` - unique items
              * `4` - gems

https://app.swaggerhub.com/apis/Chuanhsing/poe/1.0.0#/Item

Reply:
I could make different objects for the parser and the item results...

Reply:
Ahh, yeah I was not aware of that. In that case I think it's fine to leave it as it is.